src/test/java/plugins/JobDslPluginTest.java

```diff
@@ -501,6 +502,7 @@ public void should_fail_on_missing_plugin() {
      * If the function is used, the build will be marked as unstable.
      */
     @Test @WithPlugins("config-file-provider")
+    @Ignore
```

Comment:
Please add a reason to do so. Or a deadline when this will be deleted when unfixed...